api3-whitepaper.tex

```diff
@@ -106,20 +106,20 @@ \section{Introduction}
 Decentralized oracle network solutions employ third-party oracles because it is often not feasible for the API providers to operate their own oracle nodes.
 This positions third-party oracles as expensive middlemen and forms an additional attack surface.
 To eliminate these problems and have the API providers engage in the ecosystem further, API3 data feeds will be composed of first-party oracles operated by the API providers.
-This will be made possible by Airnode, a fully-serverless oracle node that is designed to require no know-how, maintenance, or upkeep from the API provider.
+This will be made possible by Airnode, a fully-serverless oracle node that is designed to require no blockchain know-how, maintenance, or upkeep from the API provider.
 The resulting dAPIs will be cost-efficient and secure against attacks from an intermediate layer of third parties.
 
 In the case of a malfunction, the dAPI user will be able to claim compensation up to a pre-negotiated amount from the staking pool.
-Kleros~\cite{kleros:2019}, an on-chain dispute resolution protocol, will be used to decide if the claim is to be paid out based on the presented evidence.
+Kleros~\cite{kleros:2019}, an on-chain dispute resolution protocol, will be used to decide if the claim is to be paid out pursuant to the service coverage terms and conditions based on the presented evidence.
 This will incentivize stakers to actively participate in governance to ensure that dAPIs are being managed transparently and in a way that minimizes security risks.
-Successful governance---successfully monetizing dAPIs while avoiding mistakes that will result in paying out insurance claims---will be rewarded in API3 tokens, which will create a positive feedback loop that will continually improve governance.
+Successful governance---successfully monetizing dAPIs while avoiding mistakes that may result in paying out service coverage claims---will be rewarded in API3 tokens, which will create a positive feedback loop that will continually improve governance.
```

Comment:
I think "will" is more accurate here, as an outage that doesn't result in a payout won't penalize stakers

Reply:
agreed, reverted this to "will"